src/frontend/src/lib/components/ui/ButtonHero.svelte

```diff
@@ -1,13 +1,25 @@
 <script lang="ts">
+	import { getContext } from 'svelte';
 	import Button from '$lib/components/ui/Button.svelte';
+	import { HERO_CONTEXT_KEY, type HeroContext } from '$lib/stores/hero.store';
 
 	export let disabled = false;
-	export let loading = false;
 	export let testId: string | undefined = undefined;
 	export let ariaLabel: string;
+
+	const { loading } = getContext<HeroContext>(HERO_CONTEXT_KEY);
```

Comment:
Here I would either pass the property, because the button is in `ui`, OR, move the button to `hero` package/folder in another PR? WDYT?

Reply:
Let's move it afterwards.

Reply:
Done with:

- PR #3279